app/Http/routes.php

```diff
@@ -10,5 +10,7 @@
 | and give it the controller to call when that URI is requested.
 |
 */
-Route::get('/', 'HomeController@index');
+Route::get('/', 'ArticlesController@index');
 Route::get('home', 'HomeController@index');
+Route::get('articles', 'ArticlesController@index');
+Route::get('article/{id}', 'ArticlesController@show');
```

Comment:
Where cái ``id`` này cho an toàn bạn ạ :smiley:!
``Route::get('article/{id}', 'ArticlesController@show')->where('id', '^[0-9]+$');``
hoặc nếu có nhiều hơn 1 parameter thì:
```Route::get('article/{parameter_1}/{parameter_2}/{parameter_2}', 'ArticlesController@show')->where([
    'parameter_1' => 'regex',
    'parameter_2' => 'regex',
    'parameter_3' => 'regex'
]);```